src/abs.rs

```diff
@@ -47,7 +49,7 @@ fn pop_or_error(path: &mut PathBuf) -> ::std::result::Result<(), io::Error> {
 /// An absolute (not _necessarily_ [canonicalized][1]) path that may or may not exist.
 ///
 /// [1]: https://doc.rust-lang.org/std/path/struct.Path.html?search=#method.canonicalize
-pub struct PathAbs(pub(crate) PathArc);
+pub struct PathAbs(pub(crate) Arc<PathBuf>);
```

Comment:
Oh I see, you removed the type `PathArc`, but all of the types under the hood are still `Arc`. Jolly good!